IntegrationTests.MediumTrust/MediumTrustSqlBackendIntegrationTest.cs

```diff
@@ -35,8 +36,7 @@ public void MediumTrust ()
           .ToArray();
 
       var types = (from t in typeof (MediumTrustSqlBackendIntegrationTest).Assembly.GetTypes ()
-                   where t.Namespace == typeof (MediumTrustSqlBackendIntegrationTest).Namespace 
-                       && t != typeof (MediumTrustSqlBackendIntegrationTest)
+                   where (t.Namespace?.StartsWith (typeof (AdditionalFromClausesSqlBackendIntegrationTest).Namespace) ?? false)
```

Comment:
Please link second issue here, too. there is an open ticket to include the subnamespaces.

Reply:
You mean that I should link it as "depends on this issue" (+ close it)? The fix was incorporated in this PR.